extlibs/CMakeLists.txt

```diff
@@ -13,15 +10,4 @@ set(STB_ROOT "${CMAKE_CURRENT_LIST_DIR}/stb" CACHE PATH "STB directory")
 # DiffLib
 set(DIFFLIB_ROOT "${CMAKE_CURRENT_LIST_DIR}/difflib" CACHE PATH "DiffLib directory")
 
-# TinyXML
-find_package(TinyXML QUIET)
-if(NOT TinyXML_FOUND)
-    add_subdirectory(tinyxml tinyxml)
-endif()
-
 ####################################
-
-foreach(target ${EXTLIBS_TARGETS})
-    set_target_properties(${target} PROPERTIES DEBUG_POSTFIX "_d")
-    set_target_properties(${target} PROPERTIES FOLDER "SofaExtlibs")
-endforeach()
```

Comment:
Is it only related to tinyxml?

Reply:
To me, yes, as it related to https://github.com/sofa-framework/sofa/blob/f6bee9f9cd69ebd102324e0181325329f96ada94/extlibs/CMakeLists.txt#L3-L5 . 
`${EXTLIBS_TARGETS}` is defined here only with the embedded version of `tinyxml` (contains all targets linked for embedded libraries). It is then empty and should not be used in the future to me as we are moving out non-header only embedded libraries.